code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1539,24 +1539,6 @@ GLOBAL_LIST_EMPTY(features_by_species)
 
 		switch(hit_area)
 			if(BODY_ZONE_HEAD)
-				if(!I.get_sharpness() && armor_block < 50)
```

Comment:
No comments or nuthin'... I can no longer merge this, this is above my paygrade.